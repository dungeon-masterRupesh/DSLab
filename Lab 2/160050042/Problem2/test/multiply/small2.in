2 3
0 2 5 
1 5 1 
3 3
4 4 1 
1 0 5 
0 5 1 
