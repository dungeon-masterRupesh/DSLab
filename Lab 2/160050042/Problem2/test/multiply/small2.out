2 3
2 25 15 
9 9 27 
